android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -347,6 +358,12 @@ public void onHostDestroy() {
         cleanUpResources();
     }
 
+    @Override
+    protected void onDetachedFromWindow() {
```

Comment:
why did you add this ? Is this really mandatory ?

Reply:
I don't think so, but is there better way to call some code when view id unmounted ?

Reply:
OK, no problem